include 2048 bevy reset game event video (#670)
rust-gamedev/rust-gamedev.github.io: content/news/022/index.md

[ld48-mmo]: https://kuviman.itch.io/dig-world/devlog/255566/so-i-made-an-mmo-in-48-hours

### [2048: Using an event system to reset the game][2048-bevy-youtube]

![YouTube Card](2048-bevy-reset-game-events.png)

> [@chrisbiscardi] published [a video][2048-bevy-youtube] about using Bevy's event readers and writers to implement a reset game button for a 2048 clone.
> 
> The video walks through implementing a "reset game" UI button in Bevy 0.5 by taking advantage of Bevy's event system for reading and writing a ResetGameEvent. We also cover recursively despawning entities and sprites.

Reviewer: ```suggestion
[@chrisbiscardi] published [a video][2048-bevy-youtube] about using Bevy's event readers
and writers to implement a reset game button for a 2048 clone.

The video walks through implementing a "reset game" UI button in Bevy 0.5 by taking
advantage of Bevy's event system for reading and writing a ResetGameEvent. It also
covers recursively despawning entities and sprites.
```

[ld48-mmo]: https://kuviman.itch.io/dig-world/devlog/255566/so-i-made-an-mmo-in-48-hours

### [2048: Using an event system to reset the game][2048-bevy-youtube]

> ![YouTube Card](2048-bevy-reset-game-events.png)

Reviewer: ```suggestion
[![YouTube Card](2048-bevy-reset-game-events.png)][2048-bevy-youtube]
```

> [@chrisbiscardi] published [a video][2048-bevy-youtube] about using Bevy's event readers
> and writers to implement a reset game button for a 2048 clone.

Reviewer: ```suggestion
[@chrisbiscardi] published [a video][2048-bevy-youtube] about using Bevy's event
readers and writers to implement a reset game button for a 2048 clone.
```
